src/main/java/com/jcabi/http/mock/MkAnswer.java

```diff
@@ -52,6 +52,7 @@
  * @since 0.10
  */
 @Immutable
+@SuppressWarnings("PMD.TooManyMethods")
```

Comment:
@pstorch here it is explained: http://www.yegor256.com/2009/03/04/pdd.html. You can also look at other puzzles